angular/src/app/modules/review/review-detail/review-detail.component.html

```diff
@@ -173,19 +183,52 @@ <h2 (click)="refreshCurrentPage()" style="font-weight: 900; margin-left: 10px">
                 <table class="table table-hover table-bordered table-striped">
                     <thead class="head-table">
                         <tr>
+                            <th >
+                                <mat-checkbox name="selectAll" (change)="setAll($event.checked)" matTooltip="Select/DeSelect all"
```

Comment:
Should check permission to show this column for the user that have permission to perform action